src/index.ts

```diff
@@ -1,3 +1,4 @@
+import 'reflect-metadata';
```

Comment:
What's this do?

Reply:
It's for the decorators in the entities. The ✨ magic ✨ behind TypeORM for those beautiful db classes

Reply:
No clue how it works tbh